network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -279,62 +309,74 @@ class SanityQuery(NetworkConfigQuery):
     """
 
     def has_conflicting_policies_with_same_order(self):
-        if self.config.type != NetworkConfig.ConfigType.Calico:
+        """
+        Check if there are Calico policies with the same order but conflicting rules.
+        :return: (bool result, conflict str)
+        :rtype: (bool, str)
+        """
+        if NetworkLayerName.K8s_Calico not in self.config.layers:
             return False, ''
-        if len(self.config.sorted_policies) <= 1:
+        calico_policies = self.config.layers[NetworkLayerName.K8s_Calico].policies_list
+        calico_policies = [policy for policy in calico_policies if isinstance(policy, CalicoNetworkPolicy)]
```

Comment:
I wonder if we should also report conflicts between K8s policies and Calico policies with `order==1000`

Reply:
Will open a separate issue for this